### PR TITLE
Cloud Native Chapter 8

### PR DESCRIPTION
**Pull request by ardalis**
## Summary

Chapter 8 files.

### CODE REVIEW

**Review by Thraka — APPROVED** [review]

**Comment by ardalis** [comment]
:shipit:

File: docs/architecture/cloud-native/identity-server.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: IdentityServer
```
Comment (mairaw) [inline docs/architecture/cloud-native/identity-server.md]: are there any other keywords that you can add to these articles with very short titles?

**Review by mairaw — APPROVED** [review]
I've made some small edits. Please take a look and see if it's good to merge @ardalis.

AAD is not allowed to be used in our content. Make sure you don't use that anywhere else in the book.

File: docs/architecture/cloud-native/identity-server.md (hunk 1 above)
Comment (ardalis) [reply docs/architecture/cloud-native/identity-server.md]: We could say "for cloud native applications" if you like.

Comment (mairaw) [reply docs/architecture/cloud-native/identity-server.md]: Works for me. Whatever if you feel will help customers find this content.

**Comment by ardalis** [comment]
All checks pass - :shipit:

**Pull request merged**